### DISCUSSION TITLE
Download PDF

### DISCUSSION THREAD

**Discussion by ereieruoi** [aclap-dev/video-downloadhelper#120] [Help and Q&A]
### Your question

I was downloading PDFs from websites, now the downloadhelper extension has updated and I can't do it anymore.

**Comment by GrampaWildWilly** [comment]
You are correct.  VDH used to be able to download .zip files, too.  That's a feature that has also gone by the boards.  It's a side effect of the download engine now being ffmpeg.  It just doesn't handle those file types.

Fortunately, the browser itself does an excellent job of downloading both those types of files, so that's what you should use from now on.

**Comment by ereieruoi** [comment]
How to download PDFs like before via VDH?
Is there a configuration in the VDH that resolves this issue? I have the license verified.
![image](https://github.com/aclap-dev/video-downloadhelper/assets/160261945/3731fe0d-78c5-4c16-bac1-63c12d426e28)

**Reply by GrampaWildWilly** [reply]
I don't think you grasped what I said.  This has nothing to do with being licensed or VDH settings.  The feature is gone from VDH.  It no longer does what it used to do.  Quit trying to make VDH download PDFs (or zips).  It no longer does it.  Forget about it.  Use the browser itself instead, not VDH.
